back/src/main/java/com/baba/back/common/domain/BaseEntity.java

```diff
@@ -0,0 +1,28 @@
+package com.baba.back.common.domain;
+
+
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.MappedSuperclass;
+import java.time.LocalDateTime;
+import lombok.Getter;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@Getter
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+public class BaseEntity {
+
+    @CreatedDate
+    private LocalDateTime createdDate;
+
+    @LastModifiedDate
+    private LocalDateTime updatedDate;
+
+    private boolean deleted = Boolean.FALSE;
+
+    public void updateDeleted() {
+        deleted = !deleted;
```

Comment:
```suggestion
        this.deleted = !this.deleted;
```
저는 this를 붙여주는 것을 선호합니다.
필드 변수임을 확실하게 알 수있거든요

Reply:
넵 앞으로 필드 변수에는 무조건 this 붙여요!